core/src/components/badge/badge.common.vars.scss

```diff
@@ -26,3 +26,9 @@ $badge-font-size: dynamic-font($badge-baseline-font-size);
 
 /// @prop - Font weight of the badge
 $badge-font-weight: bold;
+
+/// @prop - Font size of the badge hint
+$badge-hint-baseline-font-size: 8px;
+
+/// @prop - Font size of the badge hint
+$badge-hint-font-size: dynamic-font($badge-hint-baseline-font-size);
```

Comment:
Since these are for native usage only, create a `badge.native.vars.scss` file and move those tokens there. That way ionic isn't importing extra weight.

Reply:
Fixed!